snap/info.go

```diff
@@ -654,6 +654,10 @@ type SlotInfo struct {
 	Label     string
 	Apps      map[string]*AppInfo
 	Hooks     map[string]*HookInfo
+
+	// HotplugDeviceKey is the key of the associated device; it's empty for
```

Comment:
The comment is stating the code. Any DeviceKey is _always_ the _Key_ of the _Device._ Such comments are worse than no comment because it forces the reader to parse an english sentence looking for useful data, but there's none.

A better comment might be:
```
// HotplugKey is a unique key built by the slot's interface using properties of a
// hotplugged so that the same slot may be made available if the device is reinserted.